docs-src/concepts/what-is-worker-versioning.md

```diff
@@ -8,127 +8,118 @@ tags:
+
+For example, if you have a Workflow that completes within a day. A good strategy is to assign a new Build ID to every new Worker build and add it as the new overall default in the version sets.
+
+Because your Workflow completes in a day, you know that you won't need to keep older Workers running for more than a day after you deploy the new version (assuming availability).
+
+Removing old versions from the sets isn't necessary. Leaving them doesn't cause any harm.
+
+You can apply this technique to longer-lived Workflows too; however, you may need to run multiple Worker versions simultaneously while open Workflows complete.
+
+#### Deploy code changes to Workers
+
+The feature also allows you to implement compatible changes to or prevent a buggy code path from executing on currently open Workflows. You can achieve this by adding a new version to an existing set and defining it as _compatible_ with an existing version, which shouldn't execute any future Workflow Tasks. Since the new version processes existing [Event Histories](/workflows/#event-history), it must adhere to the usual [deterministic constraints](/workflows/#deterministic-constraints), and you may need to use one of the [versioning APIs](/workflows/#workflow-versioning).
+
+Moreover, this feature enables you to make incompatible changes to Activity Definitions in conjunction with incompatible changes to Workflow Definitions that use those Activities. This functionality works because any Activity that a Workflow schedules on the same Task Queue only gets dispatched to Workers compatible with the Workflow that scheduled it.
+If you want to change an Activity Definition's type signature while creating a new incompatible Build ID for a Worker, you can do so without worrying about the Activity failing to execute on some other Worker with an incompatible definition. The same principle applies to Child Workflows.
+
+:::tip
+
+Public facing Workflows on a versioned Task Queue shouldn't change their signature as it contradicts the purpose of Workflow-launching Clients remaining unaware of changes in the Workflow Definition. If you need to change a Workflow's signature, use a different Workflow Type or a completely new Task Queue.
+
+:::
+
+:::note
+
+If you schedule an Activity or a Child Workflow on _a different_ Task Queue from the one the Workflow runs on, the system doesn't assign a specific version. This means if the target queue is versioned, they run on the latest default, and if its unversioned, they operate as they would have without this feature.
+
+:::
+
+**Continue-As-New and Worker Versioning**
+
+By default, a versioned Task Queue's Continue-as-New function starts the continued Workflow on the same compatible set as the original Workflow.
+
+If you continue-as-new onto a different Task Queue, the system doesn't assign any particular version. You also have the option to specify that the continued Workflow should start using the Task Queue's latest default version.
```

Comment:
The behavior here changed a bit, but I'll reword it in my main PR after we merge this